spec/rails_helper.rb

```diff
@@ -7,6 +7,8 @@
 require 'rspec/rails'
 require 'support/factory_girl'
 require 'webmock/rspec'
+require 'shoulda/matchers'
+
```

Comment:
It is explained in commit 4f249fb. Please, let me know if this is not clear enough.

> shoulda matchers is not required in the Gemfile, and it is required in
the `spec_helper` because the matchers were not identified otherwise.
It is the recommended way to get over this issue.